tests/integration/bootstrap/FeatureContext.php

```diff
@@ -80,4 +80,18 @@ public function disableDataImport()
 		$config = \Kohana::$config->load('features');
 		$config->set('data-import.enabled', false);
 	}
+
+	/** @BeforeScenario @cdnEnabled */
+	public function enableCdn()
```

Comment:
I can't recall why it didn't work; only that it didn't. @rjmackay may be able to remember and explain.